app/Nova/Actions/SyncEngagePurchaseRequestToQuickBooks.php

```diff
@@ -106,8 +105,7 @@ public function handle(ActionFields $fields, Collection $models)
                 $line->LinkedTxn?->TxnType === 'ReimburseCharge' &&
                 $line->LinkedTxn?->TxnId === $fields->quickbooks_reimburse_charge_id
             ) {
-                $line->Amount = $envelope->amount;
```

Comment:
No longer modifying amounts here - there's validation in this PR to ensure the billable expense amount matches the Engage request approved amount. Longer term we will probably need to add support for differing amounts, but not like this.